compiler/apps/playground/components/Editor/Output.tsx

```diff
@@ -245,13 +245,34 @@ function Output({store, compilerOutput}: Props): JSX.Element {
     }
   }
 
+  const filteredTabs = useMemo(() => {
```

Comment:
Do we need this memoization? The compiler is enabled for this app

Reply:
Also, doing these here means that `tabify` is still doing the work to render out `<TextTabContent/>` components for each tab, on every source change. Then we're just ignoring most of them in the case of the filtered view.

Instead, we could skip the tabify work altogether. Maybe something like

```js
for (const [passName, results] of compilerOutput.results) {
  if (store.showInternals === false && (passName !== 'Output' || passName !== 'SourceMap' )) {
    continue;
  }
  // ...
}
```